Contributor Documentation/LSP Extensions.md

```diff
@@ -490,6 +490,60 @@ Added field:
  * Clients may ignore this parameter and add the message to the global log
  */
 logName?: string;
+
+
+/**
+ * If specified, allows grouping log messages that belong to the same originating task together instead of logging
+ * them in chronological order in which they were produced.
+ *
+ * LSP Extension guarded by the experimental `structured-logs` feature.
+ */
+structure?: StructuredLogBegin | StructuredLogReport | StructuredLogEnd;
+```
+
+With
+
+```ts
+/**
+ * Indicates the beginning of a new task that may receive updates with `StructuredLogReport` or `StructuredLogEnd`
+ * payloads.
+ */
+export interface StructuredLogBegin {
+  kind: 'begin'
+
+  /**
+   * A succinct title that can be used to describe the task that started this structured.
+   */
+  title: string;
```

Comment:
If we have title it feels a little odd to have a log message at all (though I realize that's currently being used in preparation). I think I'd prefer either having title and then mentioning that the message is ignored completely, or only having message and no title. For end, might make sense to add duration in there specifically (and also state that the message is ignored - the "finished" we have today could be its own log message)? Then completely ignore both in the unstructured case.